content/pages/FAQ.rst

```diff
@@ -15,7 +14,13 @@ How are topics chosen?
 Most session topics are scheduled the morning of the event and are provided by
 the attendees.
 
---------------
+What are Lightning Talks?
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Lightning talks are five minute presentations on any topic. At barcamp,
+the first session will be just lightning talks in the barcamp style;
```

Comment:
I thought only the first session would be lightning talks. We would have the first session be just lightning talks, and then all other sessions be regular. Am I mistaken?